rule-packs/sophos-endpoint-security

```diff
@@ -0,0 +1,166 @@
+#start-sophos-endpoint-misconfigurations
+[
+  {
+    "name": "sophos-misconfigured-endpoints-1",
+    "description": "This will query for endpoints that do not have sophos edr installed.",
```

Comment:
'that do not have the Sophos EDR Agent installed."

Reply:
Adjusting